student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'Person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission: false, name: '')
+    super(age, parent_permission: parent_permission, name: name)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permission
+  end
```

Comment:
- [ ] Good job adding the class `Student` that inherits from the class `Person`  👍 , Note that when a class inherits from another all the parent methods are accessible :accessibility:  in children classes. I noticed that you have repeat 😄  the same method `can_use_services` in the `Student` class which is already found in the parent class. To fully take advantage of the inheritance , Kindly remove it 👍